src/System.ServiceModel.Tests.Common/src/ServiceInterfaces.cs

```diff
@@ -202,3 +202,10 @@ public interface IWcfRestartService
     [OperationContract]
     String RestartService(Guid uniqueIdentifier);
 }
+
+[ServiceContract]
+public interface IWcfCustomUserNameService
+{
+    [OperationContract(Action = "http://tempuri.org/IWcfCustomUserNameService/Echo")]
+    String Echo(String message);
+}
```

Comment:
For the record, the little red icon at the end of these files is GitHub politely(?) saying it wants an end-of-line here